pkg/label/label.go

```diff
@@ -31,9 +31,9 @@ const (
 	// specific CertConfig belongs to.
 	LegacyComponent = "clusterComponent"
 
-	// RandomKeyTypeEncryption is a type of randomkey secret used for guest
-	// cluster.
-	RandomKeyTypeEncryption = "encryption"
+	// MachineDeployment label denotes which node pool corresponding resources
+	// belongs.
+	MachineDeployment = "giantswarm.io/machine-deployment"
```

Comment:
Please also put in fmt.

Reply:
https://github.com/giantswarm/fmt/pull/56